roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
+            exec:
+              command:
+              - "/usr/share/elasticsearch/probe/readiness.sh"
+            initialDelaySeconds: 5
+            timeoutSeconds: 5
```

Comment:
Created a new ticket to tune timeout value in the ES readiness probe script: https://github.com/openshift/origin-aggregated-logging/issues/370